content/monitors/faq/as-count-monitor-evaluation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: as_count() monitor evaluation changes
+kind: faq
+---
+
+## Overview
+
+Currently monitors that have the `as_count` modifier use a separate evaluation path than other monitors. In certain use-cases this this can result in unexpected behavior. We intend to migrate all monitors with `as_count` to the same evaluation path as other monitors, but this doc explains the underlying issue and will guide you through the migration process.
+
+Let's call the current evaluation path for `as_count` monitors `current_eva_path` and the new one `new_eva_path`.  
+
+The difference between those two evaluation paths is that complex monitor queries, especially those that have division or multiplication, may produce **unintended results** depending on the time aggregation function:
+
+| Path | Behavior | Expanded expression |
+|:--------|:--------|:--------|
+|**`current_eva_path`** | Aggregation function applied *before* evaluation | **(a0 + a1 + ... + a4)/(b0 + b1 + ... + b4)** |
+|**`new_eva_path`** | Aggregation function applied *after* evaluation |**(a0/b0 + a1/b2 + ... + a4/b4)**|
+
+#### Why ?
+
+Many of our customers need to be alerted when the total global error rate over a certain time-period is too high.
+
+Normal timeseries graphs like:  
+
+`sum:requests.error{*}.as_count()/ sum:requests.total{*}.as_count()` 
+
```

Comment:
Better to have space before `/`. Also need to precise time aggregation function here.